native/packages/react-native-bpk-component-theming/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-native-bpk-component-theming",
```

Comment:
Im not sure if this package should have the `-component-` naming convention. Although it does expose a React Component, it also exposes the `withTheme` HOC therefore it's more general purpose than a component.

Some other options:
- `react-native-bpk-theming`
- `react-native-bpk-theme-utils`
- ?

Reply:
I like `react-native-bpk-theming`.